src/widgets/font_description.rs

```diff
@@ -0,0 +1,117 @@
+use ffi;
+use enums;
+use glib::to_bool;
+
+use std::ffi::{CString, CStr};
+use std::str;
+
+pub struct FontDescription {
+    pub pointer: *mut ffi::PangoFontDescription
+}
+
+impl FontDescription {
+    pub fn copy(desc: &FontDescription) -> Option<FontDescription> {
+        let tmp = unsafe { ffi::pango_font_description_copy(desc.pointer) };
+
+        if tmp.is_null() {
+            None
+        } else {
+            Some(FontDescription { pointer: tmp })
+        }
+    }
+
+    pub fn equal(desc1: &FontDescription, desc2: &FontDescription) -> bool {
+        unsafe { to_bool(ffi::pango_font_description_equal(desc1.pointer, desc2.pointer)) }
+    }
+
+    pub fn set_family(desc: &mut FontDescription, family: &str) {
+        let c_family = CString::new(family).unwrap();
+        unsafe { ffi::pango_font_description_set_family(desc.pointer, c_family.as_ptr()) };
+    }
+
+    pub fn set_family_static(desc: &mut FontDescription, family: &'static str) {
+        let c_family = CString::new(family).unwrap();
+        unsafe { ffi::pango_font_description_set_family_static(desc.pointer, c_family.as_ptr()) };
```

Comment:
The contract is that the pointer stays alive for the lifetime of the object but it's freed at the end of the block.
I don't believe this optimization be worth the hassle so let's just drop this function.